docs/routing.md

```diff
@@ -26,56 +37,34 @@ This pattern is the same for all Calypso sections. If you're unfamiliar with `pa
 
 Colon-prefixed route parts (such as `:site_id` here) are called route parameters. More on them in a bit.
 
-## Middleware
+## Middlewares
 
 Each of the functions in a route definition is called a _middleware_, but in Calypso parlance, we like to refer to them as _controllers_, too.
 Each middleware takes up to two arguments, `context` and `next`. `next` is a function that if called will just invoke the next middleware in the chain, while context is a JavaScript object whose attributes you can read and write and use to communicate information from one middleware to the next. Route params are found in `context.params` (e.g. `context.params.site_id`). The last middleware in the chain -- `menus` and `sites` in our example -- usually doesn't call `next`, since, well, it's the last middleware we're calling.
 
-So what does each middleware do? Unsurprisingly, the last middleware is usually section-specific; and most often, it will render the section's primary content area (actually found in a `<div />` with `id="primary"`). The other middlewares found in this example are more generic (but specific to the `my-sites` group):
-
-* `navigation` renders the section group's sidebar to the `#secondary` `<div />`.
-* `siteSelection` parses the current route, looking out for something that looks like a URL or numeric site ID, and sets the currently selected site based on this information. You can then find it by using the `getSelectedSiteId` selector found in `state/sites/selectors`.
-* `sites` renders a site selector menu to the `#primary` `<div />` for the user to select a site, and will then redirect to that site.
-
-This wraps up the more introductory part; if you're interested in more details, keep on reading.
+So what does each middleware do?
 
-## Multi-Tree Rendering
+The last middleware before `makeLayout` is usually section-specific; and most often, it will place the section's primary view in `context.primary`. The other middlewares found in this example are more generic.
```

Comment:
Could change this to `Other middleware found in this example is more generic.` to avoid the plural.